code/modules/scavstation_shared_content/species/baxxid.dm

```diff
@@ -28,6 +28,13 @@
 		BP_EYES =     /obj/item/organ/internal/eyes/baxxid
 		)
 
+	available_cultural_info = list(
+		TAG_CULTURE =   list(CULTURE_BAXXID, CULTURE_OTHER),
+		TAG_HOMEWORLD = list(HOME_SYSTEM_STATELESS),
+		TAG_FACTION =   list(FACTION_BAXXID, FACTION_OTHER),
+		TAG_RELIGION =  list(RELIGION_OTHER, RELIGION_ATHEISM, RELIGION_AGNOSTICISM)
+	)
+
 /datum/language/baxxid
```

Comment:
This stub can be deleted.

Reply:
Removed "/datum/language/baxxid" from baxxid.dm (line 37)